parsers/nmparser/parse_run_details_test.go

```diff
@@ -127,6 +129,8 @@ func TestParseRunDetails(tt *testing.T) {
 	}
 }
 
+// This was turned off Sept 2021 but
```

Comment:
@elisarver I think this was you. Do you remember why you commented this out? Maybe I missed it, but I can't find any comment or note. I'm considering whether we should just delete it?

Reply:
I don't think so. I used Todo in all my comments.

Reply:
This was commented out way back in 516fc46 (wip: consume grd file; change bool to new type Status, 2019-09-19).  @elisarver just changed the commenting style in cd8ec1a (Wrapt, 2021-09-17)

Reply:
ah my bad. I should've clicked deeper on the git blame. Ok, I'll delete it. 2-year old commented code shall not survive this PR.